app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/analysisForm/strategyBlock/strategyBlock.jsx

```diff
@@ -72,7 +74,7 @@ export class StrategyBlock extends Component {
           fieldWrapperClassName={cx('switcher-wrapper')}
           label={intl.formatMessage(messages.autoAnalysisSwitcherTitle)}
           description={Parser(intl.formatMessage(messages.analysisStatusInfo))}
-          format={Boolean}
+          format={this.formatAnalyzerAvailabilityValue}
```

Comment:
Could you please clarify what happens here?

Reply:
We get `boolean` attributes in the strings from server. So the `true`/`false` will be `"true"`/`"false"`